lib/models/arguments.rb

```diff
@@ -24,12 +24,21 @@ def to_output
 
     arguments.concat(["--custom-transformers", *@custom_transformers]) if @custom_transformers.length.positive?
 
+    rng = Random.new
+
     set_output(
       "args",
       arguments.map do |a|
-        next a unless a.include?(" ")
+        value = a.include?(" ") ? a.inspect : a
+
+        unless value.start_with?("--")
+          name = "variable_#{rng.rand(1000..9999)}"
```

Comment:
Could this cause arguments to be overwritten if we have multiple arguments that just so happen to get the same random number here? It's a pretty low chance, but still non-zero, right? Should we do something here to check if a variable already exists with that name?

Reply:
Updated this to ensure variable names are unique! We should be ok so long as there are <900 CLI options 😉

Reply:
Isn't it < 9000? 😉

Reply:
Hah you're right - even better!